src/modules/map/store/map.store.js

```diff
@@ -60,6 +62,18 @@ export default {
          *   null, no pin will be shown.
          */
         pinnedLocation: null,
+        /**
+         * A reference to the OpenLayers map instance.
+         *
+         * @type {ol/Map}
+         */
+        instance: null,
```

Comment:
I'm not sure this is a good idea. If Vuex wraps anything it receive in some kind of deep proxy, this will kill a lot of performance.
Maybe you could add the footer module in the slot of the map module, so that you can receive the map instance through inject (see how the DrawingModule does)

Reply:
That should be doable. I'll check.